exercises/concept/cater-waiter/.meta/exemplar.py

```diff
@@ -22,11 +25,11 @@ def clean_ingredients(dish_name, dish_ingredients):
 
 
 def check_drinks(drink_name, drink_ingredients):
-    """
+    """Append "Mocktail" or "Cocktail" to `dish_name` if alcoholic ingredients are present in `drink_ingredients`.
```

Comment:
```suggestion
    """Append "Cocktail" to `drink_name` if alcoholic ingredients are present in `drink_ingredients`, otherwise append "Mocktail".
```
```suggestion
    """Append "Mocktail" or "Cocktail" to `dish_name` if alcoholic ingredients are present in `drink_ingredients`.
```

Reply:
apologies if you get double-spammed.  I had this open in two tabs, and GH ... did **not** react well.